creating_gs_app.md

```diff
@@ -59,10 +73,10 @@ rosjava generated files into the right location:
 
     you@machine:~ $ mkdir -p $HOME/.m2
     you@machine:.m2 $ cd $HOME/.m2
-    you@machine:.m2 $ ln -s $BUILD_PATH/devel/share/maven repository
+    you@machine:.m2 $ ln -s $BUILD_PATH/devel/.private/ff_msgs/share/maven repository
```

Comment:
this did not change, it's still devel/share/maven

this runs frequently and uses it..
https://github.com/nasa/isaac/blob/v0.2.6/scripts/docker/build_apk.Dockerfile#L46

Reply:
At some point when I was trying to get this to work I didn't have `/maven` in my `/devel/share` directory. I will have to retrace my steps to figure out why. I have developed a guest science app that logs coordinates from the simulator using the other path so it's a functional alternative for now.